### PR TITLE
Feature: Header IDs + anchor helper

### PR DESCRIPTION
**Pull request by gerardo-rodriguez**
This PR will address #7. 

cc: @tylersticka @erikjung @saralohr @nicolemors

### CODE REVIEW

File: src/templates/drizzle/item.hbs [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
   {{!-- Header, labels --}}
   <div class="drizzle-u-marginEnds">
-    <h2 class="
-     drizzle-u-inlineBlock
-     drizzle-u-alignMiddle
-     drizzle-u-textNoWrap
-     drizzle-u-padRight">
-      {{data.name}}
-    </h2>
+    <a href="#{{toSlug data.name}}"
```
Comment (tylersticka) [inline src/templates/drizzle/item.hbs]: Any reason the anchor is outside of the `<h2>` rather than inside?

Comment (gerardo-rodriguez) [inline src/templates/drizzle/item.hbs]: Oops, bahahaha!!! Good catch. 😅

**Comment by tylersticka** [comment]
Awesome! LGTM 👍

**Comment by erikjung** [comment]
What's the purpose of the `u-hiddenLink` utility?

Never mind, I see now.

**Comment by gerardo-rodriguez** [comment]
> What's the purpose of the u-hiddenLink utility?

@erikjung It creates a _stealth_ link. 😁  The link will only visually display as a link in the `:hover`, `:focus` or `:active` state. 

## default state:

screen shot 2016-04-21 at 4 23 31 pm

## `:hover`, `:focus` or `:active` state:

screen shot 2016-04-21 at 4 23 52 pm

File: src/assets/drizzle/styles/utils/link.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.u-hiddenLink:not(:hover, :focus, :active) {
```
Comment (erikjung) [inline src/assets/drizzle/styles/utils/link.css]: Is the desire here to make the link not appear as a link except for when hovering?

If so, then 👍, but I'd change the name to `u-linkHidden` to better align with he SUIT link helpers that we already have in use: https://github.com/suitcss/utils-link/#available-classes

**Comment by erikjung** [comment]
@mrgerardorodriguez One suggestion about the utility name, but otherwise LGTM 👍

**Pull request merged**